src/codegen/codegen_opengl.cc

```diff
@@ -0,0 +1,138 @@
+/*!
+ *  Copyright (c) 2017 by Contributors
+ * \file codegen_opengl.cc
```

Comment:
Add a specific comment that we are targeting subset of OpenGL that is working for WebGL2,(no compute shaders)

Reply:
Added.